.jenkins/pytorch/macos-common.sh

```diff
@@ -28,7 +28,7 @@ else
     numpy=1.18.5 \
     pyyaml=5.3 \
     setuptools=46.0.0 \
-    cmake=3.19 \
```

Comment:
This is from https://github.com/pytorch/pytorch/pull/84682, which needs to be landed before https://github.com/pytorch/test-infra/pull/675